lib/mix/lib/mix/tasks/xref.ex

```diff
@@ -191,4 +200,100 @@ defmodule Mix.Tasks.Xref do
   defp excluded?(module, func, arity, excludes) do
     MapSet.member?(excludes, module) or MapSet.member?(excludes, {module, func, arity})
   end
+
+  ## Callers
+
+  defp do_callers(filter) do
+    each_source_entries(&source_calls_for_filter(&1, filter), &print_calls/2)
+  end
+
+  defp source_calls_for_filter(source, filter) do
+    runtime_dispatches = source(source, :runtime_dispatches)
+    compile_dispatches = source(source, :compile_dispatches)
+    dispatches = merge_dispatches(runtime_dispatches, compile_dispatches)
+
+    for {module, func_arity_lines} <- dispatches,
+        {{func, arity}, lines} <- func_arity_lines,
+        filter.({module, func, arity}),
+        do: {module, func, arity, lines}
```

Comment:
Can you sort the lines here instead of down below? The only reason is because --unreacheable also sorts the lines after filtering and not while printing. :)